src/app/pages/storage/modules/pool-manager/components/pool-manager-wizard/steps/1-general-wizard-step/general-wizard-step.component.ts

```diff
@@ -32,34 +27,11 @@ export class GeneralWizardStepComponent implements OnInit {
     encryption: [false],
     encryptionStandard: ['AES-256-GCM', Validators.required],
     allowNonUniqueSerialDisks: [false],
-    allowDisksFromExportedPools: [[] as string[]],
+    allowExportedPools: [[] as string[]],
```

Comment:
You can move this form and `connectWarningsToStore()` to `pool-warnings` component too.